docs/server/commands.rst

```diff
@@ -167,13 +167,41 @@ clear_stats
 Clear stats cache data.
 
 Make use of :djadmin:`clear_stats` in cases where you want to remove all stats
-data.  Such a case may be where you want to recalculate stats after a change
+data. Such a case may be where you want to recalculate stats after a change
 to checks or wordcount calculations.  While it should be fine to run
 :djadmin:`refresh_stats` or :djadmin:`calculate_checks`, by first running
 :djadmin:`clear_stats` you can be sure that the stats are calculated from
 scratch.
 
 
+.. django-admin:: flush_cache
+
+flush_cache
+^^^^^^^^^^^
+
+.. versionadded:: 2.8.0
+
+Flush cache.
+
+.. warning:: You must first **stop the workers** if you flush `stats`
+   or `redis` cache.
+
+Use the :option:`--default` to flush `default` cache (django templates, project
+permissions).
+Use the :option:`--redis` to flush all data contained in `redis` cache: pending
+jobs, dirty flags, revision (it will be restored after), all data from queues.
```

Comment:
?

You mean that we already have a `--rq` global option? Then call it `--rqdata` or `--redis-queue`, but I find Redis confusing since we're removing all data from Redis right?